testsuite/MDAnalysisTests/core/test_groups.py

```diff
@@ -576,7 +576,7 @@ class TestGroupBy(object):
     # tests for the method 'groupby'
     @pytest.fixture()
     def u(self):
-        return make_Universe(('types', 'charges', 'resids'))
```

Comment:
Why did you change it from "types" to "segids"? Just aesthetics?

Reply:
I changed it because "type" and "charges" match one to one, for example TypeA matches only charge -1.5 . So, to keep the structure of the tests consistent, I changed "types" to "segids"

Reply:
If it matches 1 to 1, then there are bugs you will not catch. You want the attributes in your test to match as little as possible to each level of grouping is different from the other.

Reply:
I misread, so yes changid *to* 'segid' was the right thing to do.